nuxt-frontend/tailwind.config.js

```diff
@@ -34,10 +34,22 @@ module.exports = {
         black: {
           core: "#000000",
           900: "#3d3d3d",
+          87: "#000000DE",
+          60: "#00000099",
         },
         white: "#ffffff",
+        "white-core": {
+          80: "#FFFFFFCC",
+          60: "#FFFFFF99",
+          90: "#FFFFFF14",
+        },
         pink: {
+          0: "rgba(242, 112, 156, 0)",
           300: "#f2709c",
+          40: "rgba(242, 112, 156, .04)",
+          80: "rgba(242, 112, 156, .08)",
+          90: "rgba(242, 112, 156, .8)",
+          16: "rgba(242, 112, 156, .16)",
```

Comment:
you can use https://rgbacolorpicker.com/rgba-to-hex to get hex value of this colors